core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/export/localfile/LocalFileExporter.java

```diff
@@ -38,7 +47,22 @@ public void add(final String filePath, final Iterable<?> results) throws Operati
     }
 
     @Override
-    public Iterable<?> get(final String key) throws OperationException {
-        throw new UnsupportedOperationException("Getting export from local file is not supported");
+    public Iterable<String> get(final String filePath) throws OperationException {
+        Iterable<String> linesFromFile;
+
+        InputStream fileToBeRead;
+        final Path path = Paths.get(filePath);
+        try {
+            if (path.toFile().exists()) {
+                fileToBeRead = FileUtils.openInputStream(new File(filePath));
+            } else {
+                fileToBeRead = StreamUtil.openStream(LocalFileExporter.class, filePath);
```

Comment:
It would be better to use one of `FileUtils` or `StreamUtil` in this class and not both.